src/pages/listing/ListingCarousel.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Carousel, CarouselCaption, CarouselControl, CarouselItem } from 'reactstrap';
+
+import LazyImage from 'components/shared/LazyImage';
+
+interface Props {
+  photos: string[];
+}
+
+// TODO: add a ref so that keyboard can be used immediately when carousel opens
+// see: https://github.com/akiran/react-slick/issues/738
+
+const ListingCarousel = ({ photos }: Props) =>  {
+  const [index, setIndex] = React.useState<number>(0);
+  const next = () => setIndex((index + 1) % photos.length);
+  const previous = () => setIndex((index - 1 + photos.length) % photos.length);
+  return <Carousel activeIndex={index} next={next} previous={previous}>
```

Comment:
`<Carousel>` has a prop `interval={false}` that turns off the autoscrolling allowing users to click through photos at their own pace. tried `autoPlay={false}` but it didn't work =/